src/popup/settings.js

```diff
@@ -195,12 +196,12 @@ export default class Settings {
   }
 
   /**
```

Comment:
Function to enable sending messages to the inject script for tasks that may be run automatically.